website/docs/plugin/framework/handling-data/blocks.mdx

```diff
@@ -0,0 +1,7 @@
+---
+page_title: 'Plugin Development - Framework: Handling Data - Blocks'
+description: >-
+  Blocks.
+---
+
+# Blocks
```

Comment:
We can borrow some content from the migration guide or Terraform's language pages to describe these. 👍

Reply:
Have done exactly that 😀